packages/realm-server/main.ts

```diff
@@ -276,6 +276,24 @@ let autoMigrate = migrateDB || undefined;
       registrationSecretDeferred.fulfill(
         message.substring('registration-secret:'.length),
       );
+    } else if (
+      typeof message === 'string' &&
+      message.startsWith('execute-sql:')
```

Comment:
Initially there was a `registrationSecretDeferred` check but I have removed it, as this will cause other test files not able to trigger SQL commands at the realm server. it only works at registration with/without token tests

https://github.com/cardstack/boxel/commit/9a2fe4067d9d81c934d0ea00b018dd8ca1d1f1a4#diff-6c1f1880fe0e02056169bfe691ccd48a3b6569200a39eeb65292ebefd78b195a